app/views/hyrax/transfers/index.html.erb

```diff
@@ -1,7 +1,13 @@
-<h1><%= t("hyrax.dashboard.transfer_of_ownership") %></h1>
+<% provide :page_header do %>
```

Comment:
@jcoyne can you give a word here why `provide` instead of `content_for`?  Might not downstream consumers wish to further append to `:page_header`?